sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_text_analytics_client.py

```diff
@@ -260,6 +260,10 @@ def recognize_pii_entities(  # type: ignore
             be used for scoring, e.g. "latest", "2019-10-01". If a model-version
             is not specified, the API will default to the latest, non-preview version.
         :keyword bool show_stats: If set to true, response will contain document level statistics.
+        :keyword domain_filter: Filters the response entities to ones only included in your specified domain.
+            I.e., if set to 'PHI', will only return entities in the Personal Healthcare Information domain.
+            Currently only supports 'PHI'. See https://aka.ms/tanerpii for more information.
+        :paramtype domain_filter: str or ~azure.ai.textanalytics.PiiEntityDomainType
```

Comment:
Do we need the same :versionadded: directive here?

Reply:
No, they had it on v3.1-preview.1, just never supported it